index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="https://kit.fontawesome.com/f41acb8d77.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+          <!-- Header section -->
```

Comment:
- [ ] [OPTIONAL] Please, consider removing all comments to make your code cleaner 👍

Reply:
Ok will do that before merging  and thanks for reviewing my code @elmar8287

Reply:
> Ok will do that before merging and thanks for reviewing my code @elmar8287

👍